dds_web/templates/mail/mail_base.html

```diff
@@ -1,6 +1,23 @@
 <!doctype html>
 <html>
-
+  <script>
+    function copyTextProjectRelease(copyOption) {
+      // Get the text field
+      let copyText;
+      if (copyOption == 1){
+        copyText = document.getElementById("ProjectListing").innerHTML;
+      }
+      else if (copyOption == 2){
+        copyText = document.getElementById("ProjectDownload").innerHTML;
+      }
+       // Copy the text inside the text field
+      navigator.clipboard.writeText(copyText);
+    
+      // Alert the copied text
+      alert("Copied the text: " + copyText);
+    }  
+  </script>
```

Comment:
I like it! I'd like to change the style of it though, but as a start, just make it a bit smaller to match the command text size.